examples/sidebar_layout/app.R

```diff
@@ -0,0 +1,80 @@
+library(shiny)
+library(shiny.semantic)
+library(glue)
+
+ui <- function() {
+  shinyUI(semanticPage(
+    title = "Sidebar Layout Test",
+    theme = "spacelab",
+    sidebar_layout(
+      sidebar_panel(
+        grid_list = list(
+          layout = grid_template(default = list(
+            areas = rbind(
+              c("top_left",    "top_right"   ),
+              c("bottom_left", "bottom_right")
+            ),
+            cols_width = c("1fr", "2fr"),
+            rows_height = c("2fr", "3fr")
+          )),
+          container_style = "
+            border: 5px solid tomato;
+            background-color: indigo;
+          ",
+          area_styles = list(
+            top_right = "background-color: teal;",
+            bottom_left = "background-color: coral;"
+          )
+        ),
+        top_left = "Top left part of sidebar",
+        top_right = "Top right part of sidebar",
+        bottom_left = "Bottom left part of sidebar",
+        bottom_right = "Bottom right part of sidebar"
+      ),
+      main_panel(
+        grid_list = list(
+          layout = grid_template(default = list(
+            areas = rbind(
+              c("top_left",    "top_center",    "top_right"   ),
+              c("middle_left", "middle_center", "middle_right"),
+              c("bottom_left", "bottom_center", "bottom_right")
+            ),
+            cols_width = c("1fr", "1fr"),
+            rows_height = c("100px", "1fr", "30%")
+          )),
+          container_style = "
+            background-color: darkseagreen;
+            border: 5px solid cyan;
+            color: black
+          ",
+          area_styles = list(
+            top_left = "background-color: purple;",
+            top_right = "background-color: ivory;",
+            middle_center = "background-color: tomato;",
+            bottom_left = "background-color: mediumslateblue;",
+            bottom_right = "background-color: steelblue;"
+          )
+        ),
+        top_left = "Top left part of main panel",
```

Comment:
can we make one cell with some shiny objects to demonstrate to users that they might contain any shiny tags here?